src/System.Xml.XmlSerializer/ref/System.Xml.XmlSerializer.csproj

```diff
@@ -2,10 +2,10 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.10.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
```

Comment:
Can you use the dir.props that's under 'src/System.Xml.XmlSerializer' instead?  You may need to rebase this on latest master.